packages/codemods/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@fluentui/codemods",
+  "version": "0.1.0",
+  "description": "package to enable easy upgrades to new versions",
+  "private": true,
```

Comment:
Just FYI - this means the package won't be automatically published when you merge this PR.

At one point, we need to test these codemods in an app codebase such as OOUI

Reply:
I'll make it public.